### ISSUE TITLE
UI for Group roles and roles permissions.

### ISSUE THREAD

**Issue by ArchieDoe** [Gizra/og#238]
Hey guys!
I see that there's no UI for roles CRUD or managing roles permissions.
I've checked other issues but it seems like nobody's implementing this part.

If so, may I take this part?
Is there any ritual on how to became an OG contributor? :D

Sorry for stupid questions, I'm a newbie in contributing process in Drupal community. You may take a look at my profile, there's one D8 module port, just in case.

Thanks!

**Comment by amitaibu** [comment]
Hi @ArchieDoe, Thanks for offering the help - We'll be glad to get it!

No special ritual, apart of providing a PR (Pull request). We promise to review and guide you.

Happy contributing 😄

**Comment by amitaibu** [comment]
You could continue the effort @pfrenssen start with in https://github.com/amitaibu/og/pull/196

**Comment by ArchieDoe** [comment]
Oh, right, weird how I missed that.
Thanks!

**Comment by pfrenssen** [comment]
Yes, feel free to pick this up. I will have no time for this in the coming 2 weeks. I intend to sprint at Drupal Mountain Camp in 3 weeks time and this is on my list, but if you feel like working on this, any progress is more than welcome!

**Comment by pfrenssen** [comment]
@amitaibu maybe you can add @ArchieDoe as a collaborator on https://github.com/amitaibu/og ? Then he doesn't need to fork the PR.

**Comment by amitaibu** [comment]
done

**Comment by zerolab** [comment]
Just a note that #243 is a continuation of amitaibu/og#196

**Comment by RuZniki** [comment]
As I can see there are no progress so far. 

Does anyone have any suggestion how this UI should looks like?
Do we have existing UI for this in previous OG versions?
